src/scripts/main.js

```diff
@@ -2,7 +2,22 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const dashBoard = document.querySelector('.dashboard');
 
-// write your code here
+people.forEach(person => {
+  const dashBoardRow = document.createElement('tr');
+  const sex = person.sex === 'm' ? 'Male' : 'Female';
+  const age = person.died - person.born;
+  const century = Math.ceil(person.died / 100);
+
+  dashBoardRow.innerHTML = `
+    <td>${person.name}</td>
+    <td>${sex}</td>
+    <td>${person.born}</td>
+    <td>${person.died}</td>
+    <td>${age}</td>
+    <td>${century}</t>
+  `;
```

Comment:
also, you can use dashBoardRow.insertAdjacentHTML('afterend', and everything goes here...)